beacon-chain/rpc/validator/proposer.go

```diff
@@ -382,6 +382,11 @@ func (vs *Server) defaultEth1DataResponse(ctx context.Context, currentHeight *bi
 	if depositsTillHeight == 0 {
 		return vs.ChainStartFetcher.ChainStartEth1Data(), nil
 	}
+	// Check for the validity of deposit count.
```

Comment:
This comment doesn't make much sense or is missing some context. Please clarify.